travel_platform/travel_app/templates/travel_app/search.html

```diff
@@ -23,6 +23,28 @@
         </div>
       {% endfor %}
 
+      {% for museum in f_museums %}
```

Comment:
key in view response containing data from foursquare. Because it has little different structure than izitravel response, I put it into separate position.